### ISSUE TITLE
Is this Dead?

### ISSUE THREAD

**Issue by snoweuph** [mozilla/side-view#444]
the title is saying everything: is this project dead?

I really like this project, but there are 2 major problems I have with this extension: No extension support inside of the sideview, and the limited size of a sideview this supports, i know some people who swiched to vivaldi, becuase they need this feature on their ultrawide monitor

**Comment by bershanskiy** [comment]
@snoweuph
This extension was part of Firefox Test Pilot program, which was shut down on [January 22nd, 2019](https://medium.com/firefox-test-pilot/adios-amigo-51bec2a00072). This extension is still available on Firefox Addons, but is no longer developed.

Which extensions do you want to use with Firefox Side View? I'm one of developers of Dark Reader and currently we are contemplating whether we want to support Firefox Side View or not. We are looking for feedback here: https://github.com/darkreader/darkreader/discussions/10105

**Comment by snoweuph** [comment]
Im missing UBlock, Sponsorblock and MalSync support inside of SideView

**Comment by snoweuph** [comment]
but the Simple awnser to my question is yes it seemes. then maybe this repo should get archived?

**Comment by bershanskiy** [comment]
Could you please clarify which extensions work for you and which don't? I believe UBlock Origin and SponsorBlock should work in Side View (based on what I know a bit about their internals). If they do not, I can look into fixing them, assuming API permits.

For example, currently, I'm working on Dark Reader in Side View, but due to limitations of Firefox API, Dark Reader might not be able to track the nesting of frames within Side View and so its ON/OFF state might work slightly differently in iframe within Side View and an iframe within a regular tab.

**Comment by snoweuph** [comment]
Ublock doesnt work for me, I get Adds in there

**Comment by snoweuph** [comment]
but sposoblock works flawlessley inthere

**Comment by lfr** [comment]
Dark Reader doesn't work properly, for instance when globally disabled it remains enabled in Side-View:

![image](https://user-images.githubusercontent.com/6348118/196607883-b7044148-f15b-4d03-985c-e81116a2703e.png)

It's not just the global setting that doesn't work in Side-View, if I set a specific setting for Amazon.com pages (like dimmed), Side-View ignores it:

![image](https://user-images.githubusercontent.com/6348118/196608233-fd696743-5ccc-45f0-904b-758d6dae90eb.png)

It's as if Side-View ignored extension settings or reads extension settings from some sort of cache that doesn't get updated.

**Comment by bershanskiy** [comment]
@lfr 
> Dark Reader doesn't work properly

Yes, this is a known issue. The upcoming Dark Reader 4.9.59 will fix it by disabling Dark Reader in Firefox Sidebar. Dark Reader 4.9.60 which will release some time next month will include proper support for Sidebars. See #10105 for details.

**Comment by slimeyena** [comment]
Not an issue, just wanted somewhere to give thanks for developing this add-on. I hope it lasts, this kind of thing blew me away in vivaldi but afaik this is the only option for firefox. It's so elegantly implemented. thanks! Can't believe mozilla haven't officially implemented this yet

**Comment by Anutrix** [comment]
Seems like this is dead. Sites like Google Meet are breaking with obvious indication that it's due to a Mozilla addon: https://github.com/mozilla/side-view/issues/455.
Led to long effort of almost disabling every addon and eventual finding https://support.mozilla.org/en-US/questions/700930.

@ctbmozilla-admin Please archive this and mark it deprecated on addons.mozilla.org. Not sure if you are right person to ping(but you were only name with mozilla and admin in tag).

**Comment by jaredhirsch** [comment]
hey all (2 years later...), I guess the answer is: it's complicated 🙃  

you could argue that side-view is dead in the sense that nobody's been actively working on it for years now, but it's alive in the sense that the add-on still works, is still hosted on AMO, and open source guilt prompted me to find time to fix #455 after I heard about it. 

But yeah, I totally understand your frustration @snoweuph. There's just not enough time in the day to keep every old project humming along as we move on to new ones.

Another important sense in which this project is still alive is that it is open and anyone is welcome to contribute or fork this add-on...and I guess this means I'm volunteering to help if anyone wants to try to fix a bug that's bothering them. Just ping me in the bug and I'll try to help you with guidance and reviews, though you will have to actually do the work.

**Comment by Amparose** [comment]
> There's just not enough time in the day to keep every old project humming along as we move on to new ones.

Understandable but I am sad this extension hasn't "graduated" as it's such a useful prototype. Needs more work like Mult-Account Container support, more controls (refresh, back/forward), and proper support for other extensions. In a time where innovative browsers are popping up (and out, like Arc) this was a refreshing find for FF.
